### DISCUSSION TITLE
证件信息如何结构化提取

### DISCUSSION THREAD

**Discussion by SilkSong** [PaddlePaddle/PaddleOCR#12630] [Q&A]
我有一个识别行驶证的需求，paddleocr 的demo 可以把行驶证上所有的内容都识别了出来，但是我最终需要一种结构化的提取，比如姓名-XXX，paddleocr 有这种具体场景的模型可以使用么？
如果想要自己训练模型达到这种效果，需要一些什么技术么？有什么技术文档可以参考么?

**Comment by woodgreat** [comment]
您好，我是新学的，我用的也是paddleocr，环境是ubuntu+python，
我想向您请教一个类似的，您一定都有渡过的问题，我根据它官方社区里的用例直接运行的程序
----------------------------------------------
https://aistudio.baidu.com/projectdetail/5449611
简历信息提取（二）：HR救星！用UIE Taskflow快速完成简历信息批_副本
----------------------------------------------
做了一下，得到的结果竟是 空集：[{}]
是不是我这样的环境还要加载或设置什么模型之类的，
在别的地方贴的帖子来问都没人回应，希望您能把你运行的方法给我说一下，十分感谢！
